x/commitment/keeper/msg_server_deposit_tokens_test.go

```diff
@@ -0,0 +1,50 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	keepertest "github.com/elys-network/elys/testutil/keeper"
+	commitmentkeeper "github.com/elys-network/elys/x/commitment/keeper"
+	"github.com/elys-network/elys/x/commitment/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDepositTokens(t *testing.T) {
+	t.Skip() // TODO test keeper needs bank keeper
```

Comment:
I think it would be nice to use simapp test to reduce time on mocking dependency keepers.

Reply:
done!